packages/editor/src/package/serlo-renderer.tsx

```diff
@@ -2,26 +2,29 @@ import { createRenderers } from '@editor/editor-integration/create-renderers'
 import { editorRenderers } from '@editor/plugin/helpers/editor-renderer'
 import { LtikContext } from '@editor/plugins/edusharing-asset/ltik-context'
 import { StaticRenderer } from '@editor/static-renderer/static-renderer'
-import type { AnyEditorDocument } from '@editor/types/editor-plugins'
 import type { SupportedLanguage } from '@editor/types/language-data'
 
 import { defaultSerloEditorProps } from './config'
 import { editorData } from './editor-data'
+import { migrate } from './storage-format'
 import { InstanceDataProvider } from '@/contexts/instance-context'
 import { LoggedInDataProvider } from '@/contexts/logged-in-data-context'
 
 export interface SerloRendererProps {
   language?: SupportedLanguage
-  document?: AnyEditorDocument | AnyEditorDocument[]
+  state: unknown
   _ltik?: string
 }
 
 export function SerloRenderer(props: SerloRendererProps) {
-  const { language, _ltik } = {
+  const { language, _ltik, state } = {
     ...defaultSerloEditorProps,
     ...props,
   }
 
+  // Side note: Migrated state will not be persisted since we cannot save in static renderer view
+  const { migratedState } = migrate(state)
```

Comment:
Not sure if that is relevant for other integrations but this might increase the bundle size.

Reply:
Because of the `io-ts` npm package?

Reply:
yeah. https://bundlephobia.com/package/io-ts is down today so I could not check if it actually matters.

Reply:
Hm, we can get rid of it in case the bundle size becomes too big but it is pretty handy for these things. :) Let's test this later and if it is an issue then we can find a solution.